docs/deployment/azure/application-insights.md

```diff
@@ -9,60 +9,151 @@ ms.topic: how-to
 
-Application insights use a connection string to tell OpenTelemetry where to send the telemetry data. The connection string is specific to the instance of Application Insights you want to send the telemetry to. It can be found in the Overview page for the application insights instance.
+## Choosing how Application Insights is provisioned
 
-:::image type="content" source="../media/app-insights-connection-string.png" lightbox="../media/app-insights-connection-string.png" alt-text="Connection string placement in the Azure Application Insights portal UI.":::
+Aspire has the capability to deploy cloud resources as part of development or cloud provisioning, including Application Insights. In your Aspire app, you can decide if you want Aspire to provision an Application Insights resource at debug time, or when deploying to Azure. You can also select to use an existing Application Insights resource by providing its connection string. The connection information is managed by the resource configuration in the AppHost project.
 
-The connection string is typically supplied to the apps as an env var. There are multiple ways an env var can be set, but the easiest way to set those in .NET Aspire is in the AppHost project using the `WithEnvironment` method, That ensures that the variables are included in the manifest for deployment. The values can be changed after deployment by modifying the environment variable in the deployed environment.
+### Automatically provisioning Application Insights during development
 
-For example, for the .NET Aspire Starter Application that has two projects, it needs to be set on each project you wish to receive the telemetry from.
+With this option, an instance of Application Insights will be created for you the first time you debug an app with the Application Insights resource definition in the AppHost project. An instance of Application Insights will be created in a resource group based on the name of the AppHost project. Each different Aspire solution will create an independent Application Insights resource.
 
-```csharp
-using Aspire.Hosting;
+> Note: There is no automatic deletion of the Azure resources created this way, so you'll need to manually remove them when they are no longer needed.
+
+To use automatic provisioning you will need to specify a dependency in the AppHost project, and reference it in each project/resource that needs to send telemetry to Application Insights. The steps include:
+
+- Adding a Nuget package reference to `Aspire.Hosting.Azure.Provisioning` in the AppHost project. This will also implicitly include the `Aspire.Hosting.Azure` package.
+
+- Add [app secrets](/aspnet/core/security/app-secrets) to tell it where to create the Application Insights Resource. If using Visual Studio, right click on the AppHost project and choose *Manage User Secrets* to create and open the secrets file. Add the following keys, replacing the values as applicable:
+
+``` json
+{
+  "Azure:Location": "WestUS",
```

Comment:
nit:
```
"Azure": {
  "Location": "West US",
  "SubscriptionId": "__Replace with your subscription GUID__"
}
```

Reply:
Is this the id or the name of the region? We should be clear here.

Reply:
Idk, I just used "West US 2" when I was trying earlier.

Reply:
who would know @davidfowl?

Reply:
The location and subscription id pair are always:

- `Location`: A value from a result in this table, `az account list-locations -o table`. Best to use the `Name`, but I believe `DisplayName` works too.
- `SubscriptionId`: GUID for your sub.